### DISCUSSION TITLE
Thank you for 3.3.0

### DISCUSSION THREAD

**Discussion by agentpatience** [OpenHantek/OpenHantek6022#306] [Q&A] [answered]
I just want to send a thank you for your hard work and excellent software upgrade for Hantek 6022BE. I can tell you are very good with software engineering. I am testing the software 3.3.0 and so far everything working well on Windows 10. is there any benefit of compiling from source?

**Comment by Ho-Ro** [comment, marked as answer]
Thanks for the nice words. When you compile yourself, there is always the possibility to make small adjustments and also contribute to the project. I don't see any technical advantages otherwise, the build takes place on an up-to-date Windows server at GitHub. Apart from the official releases, I also publish regular builds as *unstable*, this status does not mean that the program is not stable, but that certain new features may still change.
